brave-lists/brave-otr.txt

```diff
@@ -0,0 +1,9 @@
+! List used by Brave for the OTR purposes
+!
+||ceta.tech.cornell.edu/
+||ceta.tech.cornell.edu/clinic
+||ceta.tech.cornell.edu/resources
```

Comment:
I think if you just have `||ceta.tech.cornell.edu` that should cover all these!

Reply:
Should be corrected now.